plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -553,6 +554,17 @@ def download_path(self):
         """
         return self.filename
 
+    def verify_size(self, location):
```

Comment:
The only reason I put it in the child classes is that `size` attribute is defined in the child classes only and is absent in the parent one. Is it a weak argument? ;)